docs/pages/frameworks/vue.md

```diff
@@ -138,3 +138,69 @@ Here is an example:
   </div>
 </sl-drawer>
 ```
+
+### Register components global
+
+If you feel tired of importing each component manually, you can add a global component register function to avoid it, more details on the [Vue doc](https://vuejs.org/guide/components/registration.html#global-registration).
+
+Here is an example:
+
+- global register function
+
+```javascript
+const getComponentName = str => {
+  const strArr = str.split('-');
+
+  return strArr
+    .map((str, idx) => {
+      return `${str[0].toUpperCase()}${str.slice(1)}`;
+    })
+    .join('');
+};
+
+export default {
+  install(app) {
+    const baseComponents = import.meta.glob('./node_modules/@shoelace-style/shoelace/dist/components/**/*.js', {
```

Comment:
This will only work if you're using Vite, also I would've assumed it would be smart enough to resolve a bare specificer and not need to the relative path to `node_modules` 🤔

Reply:
Thanks for your suggestion, I just re-pushed with a clear function that used the shoelace.js and optimized the path with the vite resolve alias.